spanner_orm/field.py

```diff
@@ -43,23 +43,43 @@ def grpc_type() -> spanner_v1.Type:
   def validate_type(value: Any) -> None:
     raise NotImplementedError
 
+  @staticmethod
+  def matches(type: str) -> bool:
+    raise NotImplementedError
+
+  @staticmethod
+  def support_length() -> bool:
+    raise NotImplementedError
+
 
 class Field(object):
   """Represents a column in a table as a field in a model."""
 
   def __init__(self,
                field_type: Type[FieldType],
                nullable: bool = False,
-               primary_key: bool = False):
+               primary_key: bool = False,
+               length: int = 0):
```

Comment:
(nit) Don't use 0 to mean "no length", in case 0 is a valid length at some point in the future. I'd do `length: Optional[int] = None` to distinguish between no length and a length of 0. It would also be nice to have a docstring explaining what None means, though you don't have to if you don't want to since that would involve documenting existing stuff that's not related to this PR.

Reply:
Done